images/build/Dockerfile

```diff
@@ -138,45 +126,6 @@ COPY images/build/installNpm.sh /tmp/scripts
 RUN chmod +x /tmp/scripts/installNpm.sh
 RUN /tmp/scripts/installNpm.sh
 
-RUN set -ex \
```

Comment:
With this PR's changes, the folder structure of various platforms would look like below:
```
root@f1467241e041:/# ls -l opt/nodejs/
total 104
lrwxrwxrwx 1 root staff    7 Dec  9 17:16 10-lts -> 10.17.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 10.1.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 10.10.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 10.14.2
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 10.16.3
drwxr-sr-x 6 root staff 4096 Dec  9 17:15 10.17.0
lrwxrwxrwx 1 root staff    7 Dec  9 17:16 12-lts -> 12.13.0
drwxr-sr-x 6 root staff 4096 Dec  9 17:15 12.13.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 12.9.1
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 4.4.7
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 4.5.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 4.8.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 6.10.3
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 6.11.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 6.17.1
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 6.2.2
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 6.6.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 6.9.3
lrwxrwxrwx 1 root staff    6 Dec  9 17:16 8-lts -> 8.16.2
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.0.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.1.4
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.11.2
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.12.0
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.15.1
drwxr-sr-x 6 root staff 4096 Dec  9 17:15 8.16.2
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.2.1
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.8.1
drwxr-sr-x 6 root staff 4096 Dec  7 18:10 8.9.4
drwxr-sr-x 6 root staff 4096 Dec  7 18:11 9.4.0
root@f1467241e041:/#
root@f1467241e041:/# ls -l opt/dotnet/sdks/
total 20
drwxr-xr-x 5 1002 sudo 4096 Apr 19  2019 1.1.14
lrwxrwxrwx 1 root root    7 Dec  9 17:01 2-lts -> 2.1.802
drwxr-xr-x 1 1004 sudo 4096 Aug 14 07:24 2.1.802
drwxr-xr-x 5 1004 sudo 4096 Aug 14 03:41 2.2.402
lrwxrwxrwx 1 root root    7 Dec  9 17:01 3-lts -> 3.1.100
drwxr-xr-x 7 1001  117 4096 Nov  9 00:06 3.0.101
drwxr-xr-x 1 1001  117 4096 Nov 17 04:43 3.1.100
root@f1467241e041:/#
root@f1467241e041:/# ls -l opt/php
total 16
lrwxrwxrwx 1 root root   15 Dec  5 00:28 5.6 -> /opt/php/5.6.40
drwxr-xr-x 8 root root 4096 Dec  5 00:28 5.6.40
lrwxrwxrwx 1 root root   15 Dec  5 00:42 7.0 -> /opt/php/7.0.33
drwxr-xr-x 9 root root 4096 Dec  5 00:42 7.0.33
lrwxrwxrwx 1 root root   15 Dec  5 00:35 7.2 -> /opt/php/7.2.24
drwxr-xr-x 9 root root 4096 Dec  5 00:35 7.2.24
lrwxrwxrwx 1 root root   15 Dec  5 00:22 7.3 -> /opt/php/7.3.11
drwxr-xr-x 9 root root 4096 Dec  5 00:22 7.3.11
lrwxrwxrwx 1 root root    6 Dec  9 17:23 lts -> 7.3.11
root@f1467241e041:/#
root@f1467241e041:/#
root@f1467241e041:/# ls -l opt/python/
total 16
drwxr-xr-x 6 root root 4096 Dec  5 00:23 2.7.17
drwxr-xr-x 6 root root 4096 Dec  5 00:17 3.6.9
drwxr-xr-x 6 root root 4096 Dec  5 00:26 3.7.5
drwxr-xr-x 6 root root 4096 Dec  5 00:21 3.8.0
lrwxrwxrwx 1 root root    5 Dec  7 18:07 latest -> 3.8.0
lrwxrwxrwx 1 root root    5 Dec  7 18:07 stable -> 3.8.0
root@f1467241e041:/#
root@f1467241e041:/#                                    
```